packages/eslint-plugin-pf-codemods/src/rules/v6/treeViewWarnSelectableStylingModifierRemoved/treeView-warn-selectable-styling-modifier-removed.ts

```diff
@@ -0,0 +1,32 @@
+import { Rule } from "eslint";
+import { JSXOpeningElement } from "estree-jsx";
+import { getFromPackage } from "../../helpers";
+
+// https://github.com/patternfly/patternfly-react/pull/10101
+module.exports = {
+  meta: {},
+  create: function (context: Rule.RuleContext) {
+    const { imports } = getFromPackage(context, "@patternfly/react-core");
+
+    const treeViewImport = imports.find(
+      (specifier) => specifier.imported.name === "TreeView"
+    );
+
```

Comment:
Could we add some logic so that it only throws this warning in the event that `pf-m-selectable` _would've_ been applied? 

For easy reference that logic was `children && (isSelectable || hasCheckbox)`

Reply:
After doing some investigation, we are not able to check it and we will have to throw a warning every time.

The reason is simple: the `children` and `hasCheckbox` props depend on the individual TreeViewDataItem objects from `data: TreeViewDataItem[]` prop of the TreeViewProps.

And since the TreeView may be used like this:
```
import {options} from "somewhere";
<TreeView data={options} ...
```
we are not able to trace it and check the condition.